bin/consented/generate-excel-local-consented.sh

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
 source default.env
-FR_ENV=local CCD_DEF_COS_URL=http://localhost:9000 CCD_DEF_CCD_URL=$npm_package_config_local_ccdUrl EXCLUDE='*-prod.json' yarn run generate-excel-consented
\ No newline at end of file
+FR_ENV=local CCD_DEF_COS_URL=$npm_package_config_local_cosUrl CCD_DEF_AAC_URL=$npm_package_config_local_aacUrl CCD_DEF_CCD_URL=$npm_package_config_local_ccdUrl EXCLUDE='*-prod.json' yarn run generate-excel-consented
```

Comment:
I guess it refers to the values in package.json, right?

Reply:
Yes